src/modules/cpu/kernel/resize.hpp

```diff
@@ -816,8 +816,8 @@ RppStatus resize_bilinear_u8_u8_host_tensor(Rpp8u *srcPtr,
         compute_dst_size_cap_host(&dstImgSize[batchCount], dstDescPtr);     // Check if the dstImgSize exceeds dst buffer size
         Rpp32f wRatio = ((Rpp32f)(roi.xywhROI.roiWidth)) / ((Rpp32f)(dstImgSize[batchCount].width));
         Rpp32f hRatio = ((Rpp32f)(roi.xywhROI.roiHeight)) / ((Rpp32f)(dstImgSize[batchCount].height));
-        Rpp32s maxHeightLimit = roi.xywhROI.roiHeight - 1;
-        Rpp32s maxWidthLimit = (roi.xywhROI.roiWidth - 1) * srcDescPtr->strides.wStride;
+        Rpp32s maxHeightLimit = roi.xywhROI.roiHeight - 2;
```

Comment:
why is this changed?

Reply:
Was a fix to match with QA test golden outputs - that are now in line with training convergence.